events/giveaways/endedGiveawayReactionAdded.js

```diff
@@ -2,6 +2,25 @@ const Discord = require("discord.js")
 module.exports = {
   async execute(giveaway, member, reaction){
```

Comment:
Since `giveaway` here is a function argument, you can name it whatever you want in this file, say `trade`, and subsequent properties would be like `trade.guildId`, `trade.channelId`, etc